src/shadowbox/server/manager_service.ts

```diff
@@ -192,6 +192,24 @@ function validateDataLimit(limit: unknown): DataLimit {
   return limit as DataLimit;
 }
 
+function validatePassword(password: string): string {
+  if (!password) {
+    throw new restifyErrors.MissingParameterError(
+      {statusCode: 400},
+      'Missing `password` parameter'
+    );
+  }
+
+  const bytesOfPassword = Buffer.byteLength(password as string, 'utf8');
```

Comment:
Why the `as string`? `password` is already a string.
```suggestion
  const bytesOfPassword = Buffer.byteLength(password, 'utf8');
```